lib/bunyan.js

```diff
@@ -333,6 +333,8 @@ function isWritable(obj) {
  *        See README.md for full details.
  *      - `level`: set the level for a single output stream (cannot be used
  *        with `streams`)
+ *      - `logOnlySetLevel`: Use level as the only level to log and not a
```

Comment:
You didn't use any attribute "logOnlySetLevel".
Should it be "logOnlyMinLevel"?